demisto_sdk/__main__.py

```diff
@@ -484,7 +484,12 @@ def generate_test_playbook(**kwargs):
     '--integration', is_flag=True, help="Create an Integration based on HelloWorld example")
 @click.option(
     '--script', is_flag=True, help="Create a script based on HelloWorldScript example")
-@click.option("--pack", is_flag=True, help="Create pack and its sub directories")
+@click.option(
+    "--pack", is_flag=True, help="Create pack and its sub directories")
+@click.option(
+    '--demisto_mock', is_flag=True, help="Add the demisto mock. Relevant only for packs.")
+@click.option(
+    '--common_server', is_flag=True, help="Add the CommonServerPython. Relevant only for packs.")
```

Comment:
this is relevant for integration/script init.
relevant for packs only if user also initialize an integration/script.

Reply:
correct. on a second thought, shouldn;t we deprecate the non pack option?